### ISSUE TITLE
Laayoune / El-Aaiún / العيون (capital of Western Sahara) needs improvements 

### ISSUE THREAD

**Issue by orangejulius** [whosonfirst-data/whosonfirst-data#302]
This looks like a super complicated place to get right, there's a lot of alt-names and a lot of history and a lot of dispute. But right now it looks like all we have is [a locality with just the original fairly sparse data from Quattroshapes](https://whosonfirst.mapzen.com/spelunker/id/421202335/), and a region that also [doesn't have much info](https://whosonfirst.mapzen.com/spelunker/id/85673941/#7/26.318/-13.278).

Some things that would be great to have captured:
- [ ] Alt names for the multiple spellings/languages for the name of the city
- [ ] Heirarchies showing the dispute between the various countries involved

More info (of course) on Wikpedia:
https://en.wikipedia.org/wiki/Laayoune
https://en.wikipedia.org/wiki/Western_Sahara

**Comment by nvkelso** [comment]
Fixing this is extremely low on my priority list.

Besides the "correctness", are we filing this to fix a test or address external feedback?

**Comment by orangejulius** [comment]
I was made aware of it from https://github.com/sebastian-meier/capitals-of-the-world/pull/1. Right now it's quite difficult to find this city in Pelias, and better alt names at least would definitely help. If it weren't the capital of a country I would also agree it's not too important.

**Comment by orangejulius** [comment]
Oh, just realized we aren't importing this into Pelias at all because there's no geometry. Simply fixing that, if possible, might be enough.

**Comment by nvkelso** [comment]
I see a geometry on that record, can you explain the "no geometry" comment more, please?

(Sebastian should probably be using Natural Earth for this type of thing, I don't think WOF is quite ready for this type of use.)

By the way, there are other capitals that are hard to find in Pelias today, like Washington, D.C.

**Comment by orangejulius** [comment]
Sorry, when I said no geometry I meant no polygon. Looking at [this GeoJSON file](https://whosonfirst.mapzen.com/data/421/202/335/421202335.geojson), it's just a point. A polygon would definitely make Pelias happier. However I definitely didn't intend this ticket to be a "drop everything and fix this immediately", I'm sorry if I gave that impression. You get to it you get to it :)

**Comment by nvkelso** [comment]
Heads up that WOF will soon have millions more POINT only places (without polygon geom). 

@orangejulius: What Pelias issue tracks importing these point only places? Can you please link it here?

**Comment by orangejulius** [comment]
Are you referring to venue records that are points? Those are fine. We don't yet import them, but we will soon (tracked in https://github.com/pelias/whosonfirst/issues/94).

And it turns out I was actually wrong, we still import administrative areas that are only points, but we discussed turning them off as mentioned in https://github.com/pelias/pelias/issues/292. We definitely prefer polygons for administrative areas since without a polygon we can't generate admin heirarchies for records from the other data sets we import (OSM, OA, Geonames). So any polygons are a huge help.

**Comment by stepps00** [comment]
A PR should be out shortly with some locality improvements in Morocco.. confirming that we'll have a new centroid and geometry for Laayoune: https://github.com/whosonfirst-data/whosonfirst-data-admin-ma/blob/stepps00/admin-updates/data/421/202/335/421202335.geojson

Since this issue has been filed, quite a some names for both the [region](https://spelunker.whosonfirst.org/id/85673941/) have been added. The [locality](https://spelunker.whosonfirst.org/id/421202335/) record needs some names, so I'll work on adding those as part of the PR.

@orangejulius it's been a while since this issue was filed.. feel free to add any other improvements that would be helpful here.

**Comment by stepps00** [comment]
I'm going to close this out once the above PR is merged. The locality should have a new Wikidata concordance and ~90 new name translation once that lands, and the region record in the initial issue comment has been superseded by this new record: https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-ma/master/data/179/666/137/1/1796661371.geojson

All of the regions in Morocco have Wikidata concordances, so name translations have been added to each. There might be some cleanup to do around hierarchies of places within disputed areas.. my preference would be to track that in a pip/spatial repo because that's likely logic that needs to be added to support complex hierarchies (unsure which repo that would belong to).

**Comment by orangejulius** [comment]
Awesome! This was always a relatively minor issue for Pelias but it does mean that we should have better results in our capital cities test suite. Thanks!
